_posts/2016-10-05-How-to-extract-ZIP-or-RAR-archives-on-your-iOS-device-for-free.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: How to extract ZIP or RAR archives on your iOS device for free
+twitter_username: pietbrauer
+categories: blog
+readtime: 3 min read
+header_image: "/img/2016-10-05-Extract-Archives-Header.jpg"
+---
+
+__The internet is full of technology that saves device storage space and bandwidth to get files to your device faster. But iOS lacks a built-in way to extract ZIP or RAR files leaving you staring at an archive instead of enjoying the latest home movie of a friend. But we got you covered, 100% for free, without ads and iPad Pro optimized.__
+
+
+Let me introduce [Documents 5](https://itunes.apple.com/us/app/documents-5-file-manager-pdf/id364901807?mt=8) to you. [Documents 5](https://itunes.apple.com/us/app/documents-5-file-manager-pdf/id364901807?mt=8) is an app developed by [Readdle](https://readdle.com), a company mostly know for [PDF Expert](https://readdle.com/products/pdfexpert5). It is a file manager unlike any other, which understands many file formats from JPEG and PNG to DOCX and Numbers. It connects to your FTP, WebDAV, Dropbox, Box and even Office 365 account.
+It truly is a one app fits all file manager. You can transfer files between storage providers, import them into other apps, organize everything in folders and add favorites. It can even play mp3 files in a playlist, a feature Apple still has not added to the stock music app, and play many video formats.
```

Comment:
'It truly is a one app-fits-all..' Maybe like this? Otherwise you have to re-read to understand, I think.